project-base/.github/workflows/run-checks-tests.yaml

```diff
@@ -35,3 +35,48 @@ jobs:
                 with:
                     name: acceptance-logs
                     path: ./var/log
+    build-whole-application-for-single-domain:
+        name: Build application and run standards checks and tests on single domain
+        runs-on: 'ubuntu-20.04'
+        steps:
+            -   name: GIT checkout branch - ${{ github.ref }}
+                uses: actions/checkout@v4
+                with:
+                    ref: ${{ github.ref }}
+            -   name: Configure application
+                run: echo 1 | ./scripts/configure.sh
+            -   name: Remove other domains than first one from configs
+                run: |
+                    # delete all domain URLs except URLs for domain with ID 1 in app/config/domains.yaml
+                    sed -i '/id: 2/{:a;Q}' app/config/domains_urls.yaml
+                    sed -i '/id: 2/{:a;Q}' app/config/domains.yaml
+
+                    # sed is removing all content from match to end of file, unfortunately it adds new line (\n) after the replacement
+                    # more info: https://stackoverflow.com/questions/50559170/delete-last-line-break-using-sed
+                    # when there stays more than one last line the build will fail because of our coding standards
+                    truncate -s -1 app/config/domains_urls.yaml
+                    truncate -s -1 app/config/domains.yaml
+            -   name: Install composer dependencies
```

Comment:
I see there are a lot of duplicated lines, could we do something with that? Is not it possible to extract the common lines? (from "Install composer dependencies" to "Upload Codeception logs to artifacts", the steps are identical for both "build-whole-application" and "build-whole-application-for-single-domain" jobs, right?

Reply:
Well, it is easier to keep this as it is for easier readability